pkg/ipam/ipam.go

```diff
@@ -104,6 +104,12 @@ func (ipam *IPAM) reserveLocalRoutes() {
 			continue
 		}
 
+		// ignore black hole route
+		if r.Src == nil && r.Gw == nil {
```

Comment:
Is `r.Src` only `nil` for blackhole routes?

Reply:
both r.sSrc and r.Gw is nil for the blackhole routes